saml2/saml2-service-provider/src/main/java/org/springframework/security/saml2/provider/service/authentication/DefaultSaml2AuthenticatedPrincipal.java

```diff
@@ -26,12 +28,15 @@
  * @author Clement Stoquart
  * @since 5.2.2
```

Comment:
Since this class is just becoming generally available in 5.4, it makes more sense to indicate that in the `@since`. Otherwise, developers may wonder why they can't find `DefaultSaml2AuthenticatedPrincipal` in 5.3, for example.

Will you please change this to `@since 5.4`?

Reply:
You are also welcome to add yourself as an author, if you like.